File: 1024java/hw20.java
import java.util.Scanner;

public class hw20 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double tax,countytax;
        tax = 0.0;
        countytax = 0.0;
        System.out.print("What is the order amount ? ");
        double price = scan.nextDouble();
        System.out.print("Which state do you live ? ");
        String state = scan.next();
        if(state.equals("Wisconsin")){
            tax = 0.055*price;
        System.out.print("What county do you live ? ");
        String county1 =scan.next();
        String county2 = scan.nextLine();
        String county = county1 + county2;
        if(county.equals("Eau Claire")){
            countytax = 0.005*price;
        }
        else if(county.equals("Dunn")){
             countytax = 0.004*price;
        }
        System.out.printf("\"The state tax is $%.2f.\"\n\"The county tax is $%.2f.\"\n\"The total tax is $%.2f.\"\n\"The total is $%.2f\"",tax,countytax,(tax+countytax),(price+tax+countytax));
        }
        else if(state.equals("Illinore")){
            tax = 0.08*price;
            System.out.printf("\"The state tax is $%.2f\"\n\"The total is $%.2f\"",tax,price+tax);
        }
        else{
            tax = 0.07*price;
            System.out.printf("\"The total is $%.2f\"",tax+price);
        }
    }
}

File: 1108/hw20.py
price = float(input("What is the order amount ? "))
state = input("What state are you in ? ")
if state == "Wisconsin":
    state_tax = 0.055*price
    county = input("What county are you in ? ")
    if county == "Eau Claire":
        county_tax = 0.005*price
    elif county == "Dunn":
        county_tax = 0.004*price
    else:
        county_tax = 0.0
    print("The state tax is $%.2f\nThe county tax is $%.2f\nThe total tax is $%.2f\nThe total is $%.2f"
          %(state_tax,county_tax,state_tax+county_tax,price+state_tax+county_tax))
elif state == "Illinoy" :
    state_tax = 0.05*price
    print("The state tax is $%.2f\nThe total is $%.2f"
          % (state_tax, price + state_tax ))
else :
    state_tax = 0.045*price
    print("The total is $%.2f"% price+state_tax)